src/config/feeds-schema.ts

```diff
@@ -34,16 +34,10 @@ export default {
             {$ref: '#/definitions/ByBitSpotFetcher'},
             {$ref: '#/definitions/BinanceFetcher'},
             {$ref: '#/definitions/GVolImpliedVolatilityFetcher'},
-            {$ref: '#/definitions/CryptoCompareHistoDayFetcher'},
-            {$ref: '#/definitions/CryptoCompareHistoHourFetcher'},
-            {$ref: '#/definitions/CryptoComparePriceFetcher'},
-            {$ref: '#/definitions/CryptoComparePriceWSFetcher'},
             {$ref: '#/definitions/PolygonIOPriceFetcher'},
             {$ref: '#/definitions/PolygonIOStockPriceFetcher'},
             {$ref: '#/definitions/PolygonIOCryptoPriceFetcher'},
             {$ref: '#/definitions/PolygonIOCurrencySnapshotGramsFetcher'},
-            {$ref: '#/definitions/IEXEnergyFetcher'},
-            {$ref: '#/definitions/BEACPIAverageFetcher'},
```

Comment:
@DZariusz let me know if these 2 are not used anymore. I removed them since I didn't find any reference to these fetchers.

Reply:
let's remove